apps/datahub/src/app/home/search/search-filters/search-filters.component.ts

```diff
@@ -53,12 +53,14 @@ export class SearchFiltersComponent implements OnInit {
     this.platformService.getMe().subscribe((user) => (this.userId = user?.id))
     this.searchConfig = (
       getOptionalSearchConfig().ADVANCED_FILTERS || [
-        'publisher',
+        'organization',
         'format',
         'publicationYear',
         'topic',
         'isSpatial',
         'license',
+        'producer',
+        'publisher',
```

Comment:
I wouldn't add these to the default advanced filters, but just keep the `organization` here, as it is currently the case.

Reply:
I hesitated in doing so, I will reverse it